### PR TITLE
Microsoft.Edge.Canary version 112.0.1705.0

### PR DESCRIPTION
**Pull request by Andrej730**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/97843)

### CODE REVIEW

**Comment by Andrej730** [comment]
Got the url from https://www.microsoftedgeinsider.com/ru-ru/download/canary

Guess it would be better if there was a way to get full offiine installer - so it would be possible for winget to auto update when new version will come but I couldn't find a way to make this link return full intallers.
`https://c2rsetup.edog.officeapps.live.com/c2r/downloadEdge.aspx?platform=Default&source=EdgeInsiderPage&Channel=Canary&language=en`

**Comment by Andrej730** [comment]
May close #90197

**Review by stephengillie — DISMISSED** [review]

File: manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Edge.Canary
+PackageVersion: 112.0.1705.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://c2rsetup.edog.officeapps.live.com/c2r/downloadEdge.aspx?platform=Default&source=EdgeInsiderPage&Channel=Canary&language=en
```
Comment (Trenly) [inline manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.installer.yaml:9]: @denelon - Can you validate this is an official URL? It doesn't seem to match `Microsoft.Edge`, `Microsoft.Edge.Beta`, or `Microsoft.Edge.Dev`

File: manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Edge.Canary
+PackageVersion: 112.0.1705.0
+PackageLocale: en-US
+Publisher: Microsoft
+PackageName: Microsoft Edge Canary
+License: Proprietary
```
Comment (Trenly) [inline manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.locale.en-US.yaml:9]: I think all of MS Edge is MIT?

cc @denelon

File: manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Edge.Canary
+PackageVersion: 112.0.1705.0
+PackageLocale: en-US
+Publisher: Microsoft
```
Comment (Trenly) [inline manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.locale.en-US.yaml:7]:
```suggestion
Publisher: Microsoft Corporation
```

File: manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Edge.Canary
+PackageVersion: 112.0.1705.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
```
Comment (Trenly) [inline manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.installer.yaml:8]: All of the other Microsoft.Edge distributions are Wix distributions, so this being different seems a bit off to me

**Review by Trenly — CHANGES_REQUESTED** [review]
This seems to be missing a lot of the metadata which is present in the Microsoft.Edge.Beta distributions like the PublisherUrl, PrivacyUrl, CopyrightUrl, and tags

File: manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.installer.yaml (hunk 4 above)
Comment (Andrej730) [reply manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.installer.yaml:8]: No idea how to identify Wix but this installer properties seem different from others
image

File: manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.locale.en-US.yaml (hunk 3 above)
Comment (Andrej730) [reply manifests/m/Microsoft/Edge/Canary/112.0.1705.0/Microsoft.Edge.Canary.locale.en-US.yaml]: Fixed

**Review by stephengillie — APPROVED** [review]

**Pull request merged**